app/Models/GamesFile.php

```diff
@@ -69,11 +69,11 @@ class GamesFile extends Model
     protected $guarded = [];
     protected $dates = ['deleted_at'];
 
-    public function gamefiletype(){
+    public function gamefiletype() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->